omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/MainSubResourceController.java

```diff
@@ -1,168 +1,205 @@
-/**

```

Comment:
don't know what happened to whole class will fix.
Change is at the end, where I add a mapping for DELETE /resource/{ruuid}/subresource to support delete all subresources in adition to existing DELETE  /resource/{ruuid}/subresource/{sruuid} same for PURGE